src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -34,6 +34,7 @@ import {
     IOUBillStackNavigator,
     IOURequestModalStackNavigator,
     DetailsModalStackNavigator,
+    ParticipantsModalStackNavigator,
```

Comment:
I'd update all instances of `Participants*` to `ReportParticipants` so that it's clearer what we are dealing with.